scripts/env.d/vars

```diff
@@ -1,5 +1,5 @@
 export DEVENVFLAVOR=""
-export DEVENVPREFIX=""
+export DEVENVPREFIX="/dev/null"
```

Comment:
Change the prefix to `/dev/null` to prevent accidental adding files to the working directory.